src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +59,44 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return Optional.of(LocalDate
+                    .of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (DateTimeException | ArrayIndexOutOfBoundsException e) {
+            return Optional.empty();
```

Comment:
let's move this return after catch block, cause we should not use try catch as if else(build logic on it)

Reply:
done